spec/CHANGELOG.md

```diff
@@ -28,6 +28,18 @@
         }
     ```
 
+  - Changed the comment sigil to `#`. (#58)
+
+  - Removed sections and introduced comment levels. (#58)
+
+    Comments starting with `##` and `###` define the outline of the file and
+    replace sections. `##` denotes group-level comments and `###` denotes
+    file-level comments. These are always standalone `Comment` entries. The
+    `Comment` node has gained the `level` field. The `comment` field on the
+    `Resource` node has been removed.
+
+  - Renamed `Symbol` to `VariantName` in the ASDL.
+
```

Comment:
Should this add something about backwards compatiblity? Is this something we do on the fluent side or on the implementations side?

If it's impl, should there be something here that suggests that impls do this?

Reply:
What do you mean by _add something_?

Reply:
Should we document how we expect implementations to deal with the update from syntax 0.4 to 0.5?

Reply:
I think we should suggest a way forward when we release 0.5. It will be based on the approach we'll take with our implementations.